### DISCUSSION TITLE
PETG issue on creality K1C

### DISCUSSION THREAD

**Discussion by lannet2000** [OrcaSlicer/OrcaSlicer#5001] [Q&A] [answered]
Hi, I'm sorry tojump in here,but I have a problem printing PETG on my new K1C and I can find a solution.
I already try to use various profiles I found on the web, but all came down with the same problem ( tracked down after many test ) the noozle head stay too low and collect filement from the printing, this generate string and hole , from very little to huge depending on the profile.
On the other side in the Creality Slicer I don't get the problem  all works fine and result are OK.
Any hint on what I can change will be appreciate.
Thanks.

**Comment by hliebscher** [comment]
do you have the same start code and the same temp settings?
my k1 max works fine with all filaments

**Reply by lannet2000** [reply]
Hi, first thank you.
Yes same setting for all test, just changed the Filament setting all the rest is the same, tested also with different brand (eSun, Overture, just got  Geeetech still have to try ) , also made a simple object 15x1x2 as reference for the test.
The problem is only on PETG setting, all the other material ( I use PLA and TPU  also the soft 85A) working good.
I'm not an experienced user, just started last year with a Ender 3 S1  and never had this problem , on that also PETG print OK.
Any help is appreciate :)

**Reply by hliebscher** [reply]
can you upload the 3mf file?

**Reply by lannet2000** [reply]
Sure but I can't upload here, not supported :(

**Reply by lannet2000** [reply]
Get from my repository
https://github.com/lannet2000/3Dprinting/blob/main/test_200x15x7.3mf

**Reply by hliebscher** [reply, marked as answer]
[test_200x15x7.txt](https://github.com/SoftFever/OrcaSlicer/files/15047801/test_200x15x7.txt)
you have wrong printer selected.  clear the overwrite settings use the machine settings ... you print to fast 

rename txt to 3mf that woks fine for me

**Reply by lannet2000** [reply]
Thanks I will fix it and try :)

**Reply by lannet2000** [reply]
Works FINE thanks a lot
